src/basic/os-util.c

```diff
@@ -98,8 +100,11 @@ int open_extension_release(const char *root, const char *extension, char **ret_p
                                 if (!image_name)
                                         continue;
 
-                                if (!image_name_is_valid(image_name))
+                                if (!image_name_is_valid(image_name)) {
+                                        log_debug("%s/%s is not a valid extension-release file name, ignoring.",
+                                                  extension_release_dir_path, de->d_name);
```

Comment:
`image_name`?

Reply:
I had that originally, but I think it's better to give the full path. It is possible that there might be more than one file with the same partial name.

Reply:
makes sense